packages/csv-parser-price/package.json

```diff
@@ -44,6 +44,7 @@
     "highland": "2.13.0",
     "lodash.mapvalues": "4.6.0",
     "lodash.memoize": "4.1.2",
+    "lodash.pick": "^4.4.0",
```

Comment:
Sorry for annoyance, but plz pin this one 👍

Reply:
uhh.. forgot, sorry